editors/emacs/Portfile

```diff
@@ -103,7 +103,7 @@ if {$subport eq "emacs-devel" || $subport eq "emacs-app-devel"} {
 
     epoch           1
     version         20201127
-    revision        1
+    revision        2
```

Comment:
The `revision` needs to be set back to 1. Adding a variant won't change anything for existing installs.